MultiPage-Clone/css/main.css

```diff
@@ -1,3 +1,370 @@
-/* Add your CSS here */
+:root {
+    font-family: 'Poppins', sans-serif;
+    font-family:'Quicksand', sans-serif;
+    --color: Dark beige: #E6DACD
+             Light beige: #F4ECE6
+             Royal blue: #0150FD
+             Black: #000
+             White: #FFF;
+}
+body {
+    font-family:'Quicksand', sans-serif;
+    margin: 20px;
+    background: linear-gradient(
+    to right,
+    #E6DACD 0%,
+    #E6DACD 50%
+    #FFF 50%
+    #FFF 100%
+    );
+}
+h1, h2, h3, h4 {
+    font-family: 'Poppins', sans-serif;
+    font-weight: 300;
+    font-size: xx-large;
+}
+h2 {
+  font-family:'Quicksand', sans-serif;
+  font-size: 1.4rem;
+}
+html, body {
+  height: 100%;
+}
+* {
+  font-family:
+  box-sizing: border-box;
+}
+main{
+  background-color: #E6DACD;
+}
+#square {
+    border: 1px solid#003cff;
```

Comment:
Do you have Prettier installed and are you linting your CSS? You might find this helpful, as there is a lot of invalid code in this PR that linting would help you catch and fix. 

It's also mandatory in later modules, so worth sorting out now. Why don't you ask someone to help you in study group in class on Saturday?